tests/snmp/test_snmp_cpu.py

```diff
@@ -51,9 +51,14 @@ def test_snmp_cpu(duthosts, enum_rand_one_per_hwsku_hostname, localhost, creds_a
         # Wait for load to reflect in SNMP
         time.sleep(20)
 
+        # Give chassis longer timeout because it has more interfaces.
+        is_chassis = duthost.get_facts().get("modular_chassis")
+        snmp_timeout = 20 if is_chassis else 5
```

Comment:
all hwsku is using the same snmp/agentx timeout config, right?

Reply:
Hi @qiluo-msft ,
Yes, with https://github.com/sonic-net/sonic-buildimage/pull/21316, all hwsku will share the 20s timeout.
But previously the other hwsku are not using the long timeout(20s) setting with SNMP tests.
That's why I limit the scope to chassis only incase hiding something on other platforms.

I'm ok if you suggest to leverage 20 to all hwskus, please let me know your suggestion.

Reply:
since that PR merged, you should use the same timeout for all hwsku. Since 20 is magic number, could you add lots of comment on how it is calculated?

Reply:
Hi @qiluo-msft , updated, make the 20s the default value and add comments.
For the timeout setting in the all the snmp requests, I think they should use the timeout setting of the snmp_fact.py by default.

Reply:
Hi @qiluo-msft , the 'retry' in snmp is actually the logic of snmp pdus,
Here the timeout is the timeout for snmp walker requests.
Per my observation, take interface snmp walker requests as example, 1 walker request has 1600 snmp pdu requests behind.

And in our production, the timeout is greater than 20s.
So with 20s timeout, we can cover the scenario well.